doc/sphinx/source/changelog.rst

```diff
@@ -4,6 +4,99 @@ Changelog
 =========
 
 
+.. _changelog-v2-6-0:
+
+v2.6.0
+------
+Highlights
```

Comment:
The title "Highlights" is not correctly formatted.

Reply:
Done